src/absa/training/distilbert.py

```
@@ -0,0 +1,45 @@
+
+def train_distilbert(train_rows, test_rows, *, epochs: int = 2, batch_size: int = 8):
+    splits = split_official_data(train_rows, test_rows)
+    tokenizer = AutoTokenizer.from_pretrained("distilbert-base-uncased")
+    model = ABSADistilBERT.from_pretrained_absa()
+    loader = DataLoader(AspectPairDataset(tokenizer, splits.train), batch_size=batch_size, shuffle=True)
+    optimiser = torch.optim.AdamW(model.parameters(), lr=2e-5)
+    model.train()
+    for _ in range(epochs):
+        for batch in loader:
+            labels = batch.pop("labels")
+            optimiser.zero_grad(); loss=torch.nn.CrossEntropyLoss()(model(**batch).logits, labels); loss.backward(); optimiser.step()
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)